app/code/Magento/Bundle/Test/Mftf/Test/AdminDisappearErrorValidateMessageFieldTest.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<!-- Test XML Example -->
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminDisappearErrorValidateMessageFieldTest">
+        <annotations>
+            <features value="Bundle"/>
+            <stories value="Create/Edit bundle product in Admin"/>
+            <title value="Assert error message for price field"/>
+            <description value="Verify error message for price field is not visible when toggle Dynamic Price is disabled"/>
+            <severity value="MAJOR"/>
+            <testCaseId value="MC-40309"/>
+            <useCaseId value="MC-30152"/>
+            <group value="bundle"/>
+        </annotations>
+
+        <before>
+            <actionGroup ref="AdminLoginActionGroup" stepKey="loginAsAdmin"/>
+        </before>
+        <after>
+            <actionGroup ref="AdminLogoutActionGroup" stepKey="logout"/>
+        </after>
+
+        <actionGroup ref="GoToSpecifiedCreateProductPageActionGroup" stepKey="createBundleProduct">
```

Comment:
Please use AdminOpenNewProductFormPageActionGroup instead as more stable and fast